src/stream/src/executor/aggregation/in_memory_value.rs

```diff
@@ -22,19 +22,18 @@ use crate::executor::aggregation::agg_impl::{create_streaming_agg_impl, Streamin
 use crate::executor::aggregation::AggCall;
 use crate::executor::error::StreamExecutorResult;
 
-/// A wrapper around [`StreamingAggImpl`], which fetches data from the state store and helps
-/// update the state. We don't use any trait to wrap around all `ManagedXxxState`, so as to reduce
-/// the overhead of creating boxed async future.
-pub struct ManagedValueState {
+/// A wrapper around [`StreamingAggImpl`], which maintains aggregation result as a value in memory.
+/// Agg executors will get the result and store it in result state table.
+pub struct InMemoryValueState {
```

Comment:
According to the very first design of aggregation from @skyzh:

> ManagedValueState: The bridge between state store and the original single value in-memory state.

The term "managed" shows that the state is persisted by this instance, which is the very contrast to the term "in-memory". 🤣 So I think this renaming might not be a good idea.

Reply:
According to offline discussion, we agree with renaming to `ValueState`.😄